frontend/src/features/projects/common/forms/UpdateInfoForm.tsx

```diff
@@ -49,7 +51,11 @@ const UpdateInfoForm = ({
         <Form.Label column md={2}>
           Assign Tier&nbsp;
           <span className="required">*</span>
-          <TooltipIcon toolTipId="tier-info" toolTip={tierTooltip} placement="right" />
+          <p></p>
```

Comment:
This can be changed if need be, it just needs to be on the page, too long for a tooltip

Reply:
Does this replace the tooltip with permanent displayed information?

Reply:
Yes